.evergreen/.evg.yml

```diff
@@ -626,8 +626,8 @@ buildvariants:
   tasks:
      - name: "test"
 
-- matrix_name: "tests-zstandard-compression"
-  matrix_spec: { compressor : "zstandard", auth: "noauth", ssl: "nossl", jdk: "jdk7", version: ["3.4", "3.6", "4.0", "latest"], topology: "standalone", os: "linux" }
+- matrix_name: "tests-zstd-compression"
+  matrix_spec: { compressor : "zstd", auth: "noauth", ssl: "nossl", jdk: "jdk7", version: ["3.4", "3.6", "4.0", "latest"], topology: "standalone", os: "linux" }
```

Comment:
No need to test against MongoDB versions that don't support zstd, which is everything but "latest".

Reply:
Now testing only against "latest".